torchvision/prototype/models/detection/ssd.py

```diff
@@ -38,11 +38,11 @@ class SSD300_VGG16_Weights(WeightsEnum):
             "map": 25.1,
         },
     )
-    default = Coco_V1
+    default = COCO_V1
 
 
 @handle_legacy_interface(
-    weights=("pretrained", SSD300_VGG16_Weights.Coco_V1),
+    weights=("pretrained", SSD300_VGG16_Weights.COCO_V1),
     weights_backbone=("pretrained_backbone", VGG16_Weights.ImageNet1K_Features),
```

Comment:
`ImageNet1K_Features` was missed

Reply:
I'm also not sure what it should be modified to, maybe `ImageNet1K_FEATURES`?

Reply:
So in the discussion with @yoshitomo-matsubara at #5088, the proposal is to have them all capitalized. My concern is that this might hurt readability in some cases and in these we should consider slight renaming. 

Another approach could be to gather all enum values in a single issue and agree upon the renaming and then split the work in multiple PRs. Again I'm flexible on how you would like to do this as long as we get an at-a-glance view of all the modifications to ensure consistency. :)